### PR TITLE
Add colors for authed players. Deprecates #1299

### PR DESCRIPTION
**Pull request by Learath2**
Not sure if a snap item is the best idea for this but it should help with not sending the data more then needed and keeping it properly updated. The colors aren't great now but we can fix that easily.

image

### CODE REVIEW

File: src/game/client/components/scoreboard.cpp [hunk 1]
```diff
@@ -444,13 +444,25 @@ void CScoreboard::RenderScoreboard(float x, float y, float w, int Team, const ch
...
+		case AUTHED_HELPER:
+			TextRender()->TextColor(0.5f, 0.0f, 0.5f, 1.0f);
+			break;
+		case AUTHED_MOD:
+			TextRender()->TextColor(0.2f, 0.2f, 0.8f, 1.0f);
+			break;
+		case AUTHED_ADMIN:
+			TextRender()->TextColor(0.0f, 1.0f, 0.0f, 1.0f);
+			break;
+		case AUTHED_NO:
```
Comment (def-) [inline src/game/client/components/scoreboard.cpp]: why even have the last case plus the default if they do nothing? either remove both or only keep authed_no.

**Comment by def-** [comment]
rest looks good

**Comment by def-** [comment]
But please make the colors brighter at least, like the team scoreboard colors.

**Comment by 12pm** [comment]
I'm not sure whether every auth level should have its own color. Might be a bit confusing and not needed?
And yes, the colors aren't great :D

**Comment by Learath2** [comment]
Maybe a brightish green for all of them?

**Comment by def-** [comment]
ok

**Comment by fokkonaut** [comment]
pls add a toggle for it

**Comment by Learath2** [comment]
image
Went with a subtler color, I still send over the exact auth level in the snap, do you think that's oversharing?

**Comment by def-** [comment]
>  Went with a subtler color, I still send over the exact auth level in the snap, do you think that's oversharing?

Keep it so that we can add it later.

Thanks for the change, lgtm

**Comment by fokkonaut** [comment]
what about a color slider in, like for laser? Then we can even disable it (make it white)

**Comment by Learath2** [comment]
@fokkonaut I'm not sure if this deserves an extra knob to tweak in settings, maybe just a toggle from console?

**Comment by fokkonaut** [comment]
ok

**Comment by fokkonaut** [comment]
?? toggle is missing

**Comment by Learath2** [comment]
Is a toggle really necessary? I choose a very subtle color and it's not
often that people are authed. I'll add one if you have to have it

**Comment by fokkonaut** [comment]
i think it would be better to have one.

**Pull request merged**